strawberry_django_plus/integrations/guardian.py

```diff
@@ -31,16 +35,16 @@ def get_object_permission_models(model: models.Model):
     )
 
 
-def get_user_or_anonymous(user: UserType) -> UserType:
+def get_user_or_anonymous(user: AbstractBaseUser) -> AbstractBaseUser:
     username = guardian_settings.ANONYMOUS_USER_NAME or ""
     if user.is_anonymous and user.username != username:
```

Comment:
This will probably solve the pyright fails

```suggestion
    if user.is_anonymous and user.get_username() != username:
```